### ISSUE TITLE
Please add compatibility with ChilloutVR

### ISSUE THREAD

**Issue by Supermagnum** [absolute-quantum/cats-blender-plugin#549]

**Comment by lautriva** [comment]
I use it frequently to convert my Vroid (=VRM) models in CVR and works without problem on my side

Maybe explain why you think CVR compatibility is broken instead of an empty issue :)

**Comment by 989onan** [comment]
dav64 is right. I use cats to help me with Neos which is wildly different from VRC

**Comment by Krislyz** [comment]
Pretty sure it is already compatible with any other software, as it takes care of the blender side of stuff and not the unity side of stuff. It's mainly created to fit VRChat's needs, but the features made for VRChat should be compatible with the rest of the major platforms' APIs.

**Comment by Retro-Jon** [comment]
I agree, better support would be great. I only use it for setting up my visemes at the moment but all other features offered seem to break compatibility with CVR. Can't really offer much detail as I'm still new to making avatars though.
